src/v2/Apps/Conversation/Components/TimeSince.tsx

```diff
@@ -85,12 +85,15 @@ export const TimeSince: React.FC<TimeSinceProps> = ({
   exact,
   ...props
 }) => {
+  if (!time) {
+    return
+  }
   return (
     time && (
       <Box {...props}>
-        <Sans size={size} color="black30">
+        <Text variant="small" color="black30">
```

Comment:
does this mean the `size` prop is no longer needed?

Reply:
@can I merge after these fixes?